.github/workflows/pki.yml

```diff
@@ -22,10 +22,10 @@ jobs:
 
     steps:
     - uses: actions/checkout@v3
-    - name: Set up Python 3.8
+    - name: Set up Python 3.9
```

Comment:
Please change the task_runner_e2e.yaml workflow as well. It runs nightly so not failed for the PR.

Reply:
done